### PR TITLE
corrected footer to exclude redirects

### PR DESCRIPTION
**Pull request by krtb**
@moxiegirl, I located the file and edited out the links that were causing the redirects we were speaking of. Please let me know if this solves the problem you were having.

### CODE REVIEW

File: pages/legal/disclaimers/disclaimers.md [hunk 1]
```diff
@@ -1,8 +1,4 @@
 **Effective Date: July 26, 2018**
 
 The Blockstack Tokens are a crypto asset that is currently being developed by
```
Comment (moxiegirl) [inline pages/legal/disclaimers/disclaimers.md]: @krtb Thanks for the PR a couple of comments.  I think you went a bit too far.  The final should read:

```
The Blockstack Tokens are a crypto asset that is currently being developed by
Blockstack Token LLC, a Delaware limited liability company, whose website can be
found at [www.blockstack.com](https://www.blockstack.com). The website you are
currently visiting [(www.blockstack.org)](www.blockstack.org) is sponsored by
Blockstack PBC, an affiliate of Blockstack Token LLC, and should not be viewed
as an offer or sale of securities.
```

This is the only section `, whose website can be found at [www.blockstack.com](https://www.blockstack.com)` that needs to be removed.

BTW, you can test your changes locally before sending your PR. The Contributing section in the README explains how to test using either `npm` or `yarn`.

**Comment by krtb** [comment]
my apologies @moxiegirl, I've re-added the rest of the text to the paragraph and only deleted the line that you had specified. And I made sure to sign the contributor's agreement the first time that I submitted a pull request, so we should be all set. Are there any other tweaks that are needed?

**Comment by moxiegirl** [comment]
@krtb Can you sign per this: 

https://github.com/blockstack/blockstack.org/pull/686#issuecomment-434372694

**Comment by krtb** [comment]
@moxiegirl, I think this time I've signed, let me know if there are any issues. When I click on the link now I get a prompt that I've already signed. I think we're all set 👍🏽

File: pages/legal/disclaimers/disclaimers.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
-Blockstack Token LLC, a Delaware limited liability company, whose website can be
-found at [www.blockstack.com](https://www.blockstack.com). The website you are
-currently visiting [(www.blockstack.org)](www.blockstack.org) is sponsored by
-Blockstack PBC, an affiliate of Blockstack Token LLC, and should not be viewed
-as an offer or sale of securities.
+Blockstack Token LLC. The website you are currently visiting
```
Comment (moxiegirl) [inline pages/legal/disclaimers/disclaimers.md]: Hi, @krtb still a bit too much trim.  We need to keep the `LLC, a Delaware limited liability company.` phrase.  Thanks for signing, almost there.

Comment (krtb) [reply pages/legal/disclaimers/disclaimers.md]: Sorry about that @moxiegirl, I've added that line back in and pushed my changes. Thanks for your patience.

**Comment by aulneau** [comment]
Hey @krtb ! Thanks for taking the time to work on this. I'm not sure that anything in this repository needs much updating from community members. If you're looking to contribute to our open source projects, please head on over to https://github.com/blockstack/blockstack-browser or https://github.com/blockstack/blockstack.js and browse the open issues.

**Comment by krtb** [comment]
Will do @aulneau, and good luck with Blockstack!

**Pull request closed without merging**